config/versions/v5.go

```diff
@@ -29,11 +29,12 @@ func (v *Version5) UpgradeExchange(_ context.Context, e []byte) ([]byte, error)
 	if err := json.Unmarshal(fsJSON, &fs); err != nil {
 		return e, err
 	}
-	if _, ok := fs["futures"]; !ok {
+	futures, ok := fs["futures"]
```

Comment:
futures -> to another name 😭 it conflicts with another package sorry.

Reply:
Is that a problem for your CI or something?
I normally rename when I've shadowed some sort of package name that's standard (e.g. `fmt`), but not really otherwise.

Fixed https://github.com/gbjk/gocryptotrader/commit/a1b0264e1

Reply:
Not me, I think it was @gloriousCode maybe? It could just be convention, but it's a fair point. Could just become less strict moving forward on this if others agree.

Reply:
It's usually important in exchange packages since they end up being local shadow vars of actually imported packages. Typically the var is not important enough to share that name, kinda like this one if it can just be changed to `f`

So it's not a strict rule, but it's usually not a huge deal to not do it as well.